### ISSUE TITLE
Structured data not present when creating Review via CLI

### ISSUE THREAD

**Issue by ratzrattillo** [woocommerce/woocommerce#24898]
**Describe the bug**
I noticed, that if i create a review on a product via cli (wp wc product_review create ), the structured data embdedded in the product page does not contain relevant information like aggregateRating or review.
If i however then use the GUI to jump to the comment section and "Update" teh comment again, without any changes, the structured data appears on the product page.

**To Reproduce**
1. Setup a new wordpress instance with woocommerce and create a product
2. Use wp wc cli product_review create to create a review
3. Go to the product page, logged in as a customer and go to the product page, which contains the cli-created review.
4. Check the structured data in the page source (Should not be complete, review and aggregate Rating should be missing)
5. Go to the comment section in the GUI, and "Update" your comment without changing anything
6. Repeat step 3 and 4, but now review and aggregate Rating should be there!

**Screenshots**

**Expected behavior**
Structured data should always be complete, regardless of comments being created via the GUI or the CLI

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront, Shop Isle, GeneratePress](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by ratzrattillo** [comment]
As you already know gtin or mpn is also always missing. Why not add something like this:
`function jl_add_mpn( $markup, $product )
	{
		$markup['mpn'] = get_post_meta($product->get_id(), "_ts_mpn", true);
		return $markup;
	}`

**Comment by claudiosanches** [comment]
Sounds like CLI/REST API is not setting the correct comment type.

**Comment by ratzrattillo** [comment]
Yes, however, when i have more than one review, it seems to work again. This might also be a logical error, that is not handling markup, when just one review is present... I also does not show the "Rating stars" when only one review is present.

**Comment by lsinger** [comment]
Duplicate of or related to #29906.

**Comment by jamesckemp** [comment]
This issue has been automatically closed as part of the backlog cleanup announced in [Refining how we manage the WooCommerce backlog](https://developer.woocommerce.com/2026/05/29/backlog-cleanup/). It met the criteria outlined there. If this still affects a current version of WooCommerce, please follow the steps in that post. A fresh report with current reproduction steps is the fastest path to triage.
